src/globus_cli/commands/flows/run/resume.py

```diff
@@ -43,16 +43,10 @@ def resume_command(
 
     specific_flow_client = login_manager.get_specific_flow_client(flow_id)
 
+    gare: GlobusAuthRequirementsError | None = None
     if not skip_inactive_reason_check:
         gare = _get_inactive_reason(run_doc)
-        if gare is not None and gare.authorization_parameters.required_scopes:
-            if not _has_required_consent(
-                login_manager, gare.authorization_parameters.required_scopes
-            ):
-                raise CLIAuthRequirementsError(
-                    "This run is missing a necessary consent in order to resume.",
-                    gare=gare,
-                )
+        check_inactive_reason(login_manager, gare)
```

Comment:
GitHub is balking at letting me reference a line of code just above this.

It looks like `specific_flow_client` is set up prior to its use in this code (unless calling `login_manager.get_specific_flow_client(flow_id)` modifies `login_manager` as a side effect).

I recommend moving this condition above that unhighlighted line since it isn't needed yet.